### PR TITLE
Animate Button on Hover

### PR DESCRIPTION
**Pull request by achen409**
Mouse in hovering over button
image

### CODE REVIEW

File: src/components/Events.jsx [hunk 1]
```diff
@@ -30,7 +30,9 @@ const Events = () => {
           />
         ))}
       </div>
-      <Button text={"See Calendar"} />
+      <div className="hover:invert">
```
Comment (SafeDuck) [inline src/components/Events.jsx:33]: im not sure why it's blue, but when you hover over it should make the text yellow and the bg white

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/components/Events.jsx (hunk 1 above)
Comment (achen409) [reply src/components/Events.jsx:33]: Won't white bg dissapear into the white background?
image

Comment (SafeDuck) [reply src/components/Events.jsx:33]: yeah that's okay, i think the current blue doesn't look good

**Comment by achen409** [comment]
image
The background turns white. The text color doesn't change on hover. Don't know how to address that.

**Pull request closed without merging**